lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -124,10 +124,11 @@ def main(argv=None):
              'converted to percentiles, as part of converting the input '
              'probabilities into realizations.')
     parser.add_argument(
-        '--predictor_of_mean', metavar='PREDICTOR_OF_MEAN',
+        '--predictor', metavar='PREDICTOR',
         choices=['mean', 'realizations'], default='mean',
-        help='String to specify the predictor used to calibrate the forecast '
-             'mean. Currently the ensemble mean ("mean") and the ensemble '
+        help='String to specify the form of the predictor used to calculate '
```

Comment:
Again not sure this is quite true for realization case

Reply:
And same below

Reply:
And in other CLIs

Reply:
A location parameter is calculated regardless of whether the predictor is the ensemble mean or ensemble realizations. Does that make sense?